board/forms.py

```diff
@@ -0,0 +1,35 @@
+from django import forms
+
+class PostForm(forms.Form):
+    """
+    A form representing the creation of a new post.
+    - `name` is the author's name
+    - `message` is the message being written in the post
+    - `photo` is the photo(s) attached to the post
+
+    None of the fields are required. However, at least either of the message or photo
+    must exist for this to be a valid post.
```

Comment:
Generally, I prefer having the attributes at the bottom of the class docstrings, and any descriptive text above the attributes.